agent/grpc-middleware/rate_limit_mappings.gen.go

```diff
@@ -11,17 +11,5 @@ var rpcRateLimitSpecs = map[string]rate.OperationType{
 	"/hashicorp.consul.dataplane.DataplaneService/GetEnvoyBootstrapParams":       rate.OperationTypeRead,
 	"/hashicorp.consul.dataplane.DataplaneService/GetSupportedDataplaneFeatures": rate.OperationTypeRead,
 	"/hashicorp.consul.dns.DNSService/Query":                                     rate.OperationTypeRead,
-	"/hashicorp.consul.internal.operator.OperatorService/TransferLeader":         rate.OperationTypeExempt,
```

Comment:
Should these be here?

Reply:
Yes, I had fixed that but somehow the fix didn't make it over to oss